### DISCUSSION TITLE
MCA Custom Skins

### DISCUSSION THREAD

**Discussion by Khavyer** [Luke100000/minecraft-comes-alive#344] [Q&A] [answered]
Hello everyone!
I was a 1.16.5 user and I added custom skins as resource pack for that version and it worked fine.
But, now, I am on 1.18.2 and the same method don't work. 
Is there something changed?

**Comment by CDAGaming** [comment, marked as answer]
See https://github.com/Luke100000/minecraft-comes-alive/wiki/Custom-Skins
